app/presenters/openid_connect_user_info_presenter.rb

```diff
@@ -75,17 +75,21 @@ def address
       street_address: street_address,
       locality: stringify_attr(ial2_data.city),
       region: stringify_attr(ial2_data.state),
-      postal_code: stringify_attr(ial2_data.zipcode),
+      postal_code: postal_code,
     }
   end
 
   def formatted_address
     [
       street_address,
-      "#{ial2_data.city}, #{ial2_data.state} #{ial2_data.zipcode}",
+      "#{ial2_data.city}, #{ial2_data.state} #{postal_code}",
     ].compact.join("\n")
   end
 
+  def postal_code
+    stringify_attr(ial2_data.zipcode)&.slice(0, 5)
```

Comment:
this approach is pretty naive. If we have spaces in our zipcodes `"   12345"` this would be wrong. I don't know how much we trust our data

Reply:
We could strip anything that isn't numeric then get the first 5? I think it _should_ be correct, but of course we can't inspect it.

Reply:
Added a plain `#strip` in a635396e14c4219b6ae5ded7fc601d6dcc3f368c